website/blog/2021-06-29-release-apache-apisix-2.7-en.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Release Apache APISIX 2.7.0"
+author: Serendipity96
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+---
+> [@Serendipity96](https://github.com/Serendipity96), Apache APISIX contributor from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+<!--truncate-->
+
+Apache APISIX 2.7.0 has been released! Welcome to download and use it.
+
+This version supports multi-language plugins, enhances the four-layer TCP proxy, and enhances Nginx configuration. More than 20 developers participated, and more than 70 pull requests have been submitted. The following is an introduction to the key features.
```

Comment:
```suggestion
This version supports multi-language plugins, enhances the four-layer TCP proxy and Nginx configuration. More than 20 developers participated, and 70 plus pull requests have been submitted. The following is an introduction to the key features.
```

Reply:
it's a bit duplicated: `"More than" 20 developers participated, and "more than" 70 pull requests have been submitted`

I think changing it to another expression could be fluent